diff --git a/node_modules/react-native-camera/package.json b/node_modules/react-native-camera/package.json
index 5e2dc63..cea2a2d 100644
--- a/node_modules/react-native-camera/package.json
+++ b/node_modules/react-native-camera/package.json
@@ -9,7 +9,8 @@
     "logo": "https://opencollective.com/opencollective/logo.txt"
   },
   "dependencies": {
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.6.2",
+	"deprecated-react-native-prop-types": "^2.3.0"
   },
   "devDependencies": {
     "@babel/runtime": "^7.3.1",
diff --git a/node_modules/react-native-camera/src/RNCamera.js b/node_modules/react-native-camera/src/RNCamera.js
index b7a271a..9408751 100644
--- a/node_modules/react-native-camera/src/RNCamera.js
+++ b/node_modules/react-native-camera/src/RNCamera.js
@@ -5,7 +5,6 @@ import {
   findNodeHandle,
   Platform,
   NativeModules,
-  ViewPropTypes,
   requireNativeComponent,
   View,
   ActivityIndicator,
@@ -13,6 +12,7 @@ import {
   StyleSheet,
   PermissionsAndroid,
 } from 'react-native';
+import { ViewPropTypes } from "deprecated-react-native-prop-types";
 
 import type { FaceFeature } from './FaceDetector';
 
